src/modules/recipes/RecipesProvider.jsx

```diff
@@ -0,0 +1,49 @@
+import { createContext, useReducer, useContext } from "react";
+
+export const RECIPE_ACTIONS = {
+  update: "update",
+  refresh: "refresh",
+  deleteAll: "delete_all",
+};
+
+export const RecipesContext = createContext();
+export const RecipesDispatchContext = createContext();
+
+export const RecipesProvider = ({ children, initialState = [] }) => {
```

Comment:
Initial state isn't used anywhere, remove it
```suggestion
export const RecipesProvider = ({ children}) => {
```